frontend/src/views/funding/FundingOverviewView.vue

```diff
@@ -25,21 +25,130 @@
+  v => isValidEndDate({ startDate: formattedStartDateFrom, endDate: v }) || 'End date must be after start date']"
+                          @click="on && on.click"
+                          @click:prepend="menuStartDateTo = !menuStartDateTo"></v-text-field>
+                      </template>
+                      <v-date-picker v-model="startDateTo" @input="menuStartDateTo = false"></v-date-picker>
+                    </v-menu>
+                  </v-col>
+                </template>
+              </v-row>
+            </v-card>
+            <v-row class="d-flex justify-end pb-3">
+              <AppButton id="reset" :primary="false" size="large" width="100px" :loading="loading" class="mr-8" @click="initialize()">Reset</AppButton>
+              <AppButton id="search" size="large" width="150px" class="mr-4" :loading="loading" @click="getFundingAgreements()">Search</AppButton>
+            </v-row>
             <h2>Funding Details</h2>
-            <v-data-table :headers="headers" :items="fundingAgreements" item-key="guid" :items-per-page="10" density="compact">
-              <template #[`item.actions`]="{ item }">
-                <v-btn v-if="item.status === 'FA Signature Pending'" variant="text" @click="this.$router.push({ name: 'funding', params: { fundingGuid: item.guid } })">
-                  <v-icon size="large">mdi-signature-freehand</v-icon>
-                </v-btn>
-                <v-btn v-if="item.status === 'FA Submitted to Ministry'" variant="text" @click="this.$router.push({ name: 'funding', params: { fundingGuid: item.guid } })">
-                  <v-icon size="large">mdi-folder-open-outline</v-icon>
-                </v-btn>
-              </template>
-            </v-data-table>
+            <v-skeleton-loader :loading="loading" type="table-tbody">
+              <v-data-table :headers="headers" :items="fundingAgreements" item-key="guid" :items-per-page="10" density="compact">
+                <template #item.startDate="{ item }">
+                  {{ format.formatDate(item?.startDate) }}
+                </template>
+                <template #item.endDate="{ item }">
+                  {{ format.formatDate(item?.endDate) }}
+                </template>
+                <template #item.status="{ item }">
+                  <span :class="getStatusClass(item?.statusCode)" class="pt-1 pb-1 pl-2 pr-2">{{ item?.status }}</span>
+                </template>
+                <template #[`item.actions`]="{ item }">
+                  <v-btn v-if="item?.status === 'FA Signature Pending'" variant="text" @click="this.$router.push({ name: 'funding', params: { fundingGuid: item.fundingId } })">
+                    <v-icon size="large">mdi-signature-freehand</v-icon>
+                  </v-btn>
+                  <v-btn v-if="['FA Submitted to Ministry', 'Active'].includes(item?.status)" variant="text" @click="this.$router.push({ name: 'funding', params: { fundingGuid: item.guid } })">
```

Comment:
{ fundingGuid: item.guid } should be { fundingGuid: item.fundingId } otherwise the link is broken

Reply:
That would def break the link, but branch has been updated to user fundingId since yesterday afternoon when i messaged you the link was working. Let me know if i'm missing something.